src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -9,16 +9,16 @@
 
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.Person;
+import seedu.address.model.card.Card;
 
 /**
- * Adds a person to the address book.
+ * Adds a card to the address book.
  */
 public class AddCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a card to the address book. "
```

Comment:
Just to check, so when you run the `add` command right now, it is adding a card instead of a person?